### DISCUSSION TITLE
Game crashes randomly

### DISCUSSION THREAD

**Discussion by Slax2317** [voxeltycoon/issues#651] [Troubleshooting]
**Describe the bug**
Game crashes randomly, even when I'm just roaming around and not doing anything memory-intensive.
All my drivers are up to date. Also this bug used to happen sometimes with the itch.io version too.

**Steps to reproduce**
Nothing special, it just happens out of the blue.

**Build version**
v0.85.0.24 steam

**Platform**
Windows 10 Pro (ver. 20H2) x64
CPU: i7-8700 @ 3.20GHz
RAM: 16,0 GB
GPU: GTX670-DC2-2GD5

**Reproducibility**
Sometimes after an hour, sometimes after 10 minutes.
(The attached log file is a gameplay of 70 minutes and it already crashed twice)

**Logs/saves**
[Player-prev.log](https://github.com/voxeltycoon/bug-tracker/files/6329065/Player-prev.log)
[m008bj937lo.zip](https://github.com/voxeltycoon/bug-tracker/files/6329071/m008bj937lo.zip)

**Comment by katepolak** [comment]
Thank you for the report. Sadly the log you have attached contains basically no information, luckily you posted the correct one on Discord so I am attaching it bellow and at the same time marking the issue as confirmed, but I suspect that we will need the crash report generated by Unity itself as even that log doesn't contain much information about what happened.

[Player-prev.log](https://github.com/voxeltycoon/bug-tracker/files/6330927/Player-prev.log)
